dotnet/src/Extensions/Planning.StepwisePlanner/Planning.StepwisePlanner.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Planning.StepwisePlanner</AssemblyName>
+    <RootNamespace>Microsoft.SemanticKernel.Planning.Stepwise</RootNamespace>
```

Comment:
Note to selves: we need to clean this up for all planners. No reason the assembly, nuget and root namespace shouldn't be identical.

Reply:
https://github.com/orgs/microsoft/projects/866/views/3?pane=issue&itemId=31762117